userspace/falco/statsfilewriter.cpp

```diff
@@ -67,30 +67,6 @@ bool StatsFileWriter::init(std::shared_ptr<sinsp> inspector, string &filename, u
 		return false;
 	}
 
-	// (Undocumented) feature. Take any environment keys prefixed
```

Comment:
Can you spend a couple of words about the reasoning behind this removal?

Reply:
yeah the `run_performance_tests.sh` was the only script that used it, so I think we could remove it :thinking: